eng/Versions.props

```diff
@@ -59,10 +59,10 @@
   <PropertyGroup Label="Automated">
     <!-- Packages from dotnet/runtime -->
     <MicrosoftExtensionsDependencyModelPackageVersion>5.0.0</MicrosoftExtensionsDependencyModelPackageVersion>
-    <MicrosoftNETCoreAppInternalPackageVersion>5.0.9-servicing.21359.8</MicrosoftNETCoreAppInternalPackageVersion>
+    <MicrosoftNETCoreAppInternalPackageVersion>5.0.10-servicing.21403.8</MicrosoftNETCoreAppInternalPackageVersion>
     <MicrosoftNETCoreAppRefPackageVersion>5.0.0</MicrosoftNETCoreAppRefPackageVersion>
-    <MicrosoftNETCoreAppRuntimewinx64PackageVersion>5.0.9</MicrosoftNETCoreAppRuntimewinx64PackageVersion>
-    <MicrosoftNETCoreBrowserDebugHostTransportPackageVersion>5.0.9-servicing.21359.8</MicrosoftNETCoreBrowserDebugHostTransportPackageVersion>
+    <MicrosoftNETCoreAppRuntimewinx64PackageVersion>5.0.10</MicrosoftNETCoreAppRuntimewinx64PackageVersion>
+    <MicrosoftNETCoreBrowserDebugHostTransportPackageVersion>5.0.10-servicing.21403.8</MicrosoftNETCoreBrowserDebugHostTransportPackageVersion>
```

Comment:
`.\restore.cmd` attempts in this branch aren't working. Is the productVersion.txt file in place at the expected location @mmitche @vseanreesermsft @lukas-lansky @wtgodbe❔

Reply:
It turns out the product was incoherent via aspnetcore in 5.0. Make the runtime dependencies 5.0.10-servicing.21412.14`

Reply:
Should I change the runtime SHAs as well❔ I suspect not (because the build doesn't use them and the next dependency update will overwrite them) but am double-checking.

Separately, how did we get incoherent❔